<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyOS</class>
 <widget class="QMainWindow" name="MyOS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyOS</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="diskInfo">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>515</width>
      <height>358</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: &lt;width&gt;px &lt;style&gt; &lt;color&gt;;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="clearTrashButton">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>173</y>
       <width>145</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear bin</string>
     </property>
    </widget>
    <widget class="QLabel" name="downloads_circle">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>234</y>
       <width>6</width>
       <height>6</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="otherDisks">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>143</width>
       <height>18</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="storage_9">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>303</y>
       <width>145</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Performance - 100%</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="storage_10">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>320</y>
       <width>145</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Condition - 100%</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>152</x>
       <y>5</y>
       <width>360</width>
       <height>349</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="movable">
      <bool>true</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Files</string>
      </attribute>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>3</x>
         <y>4</y>
         <width>18</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="usernameLineEdit_2">
       <property name="geometry">
        <rect>
         <x>25</x>
         <y>4</y>
         <width>210</width>
         <height>18</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>File's location</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="usernameLineEdit_3">
       <property name="geometry">
        <rect>
         <x>239</x>
         <y>4</y>
         <width>113</width>
         <height>18</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>Search</string>
       </property>
      </widget>
      <widget class="QTreeView" name="treeView">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>23</y>
         <width>352</width>
         <height>300</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="autoScrollMargin">
        <number>10</number>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="autoExpandDelay">
        <number>-1</number>
       </property>
       <property name="indentation">
        <number>0</number>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
       <property name="itemsExpandable">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="addButton">
     <property name="geometry">
      <rect>
       <x>253</x>
       <y>7</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="storage_13">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>337</y>
       <width>145</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Data Written - X TB</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QFrame" name="circleBar">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>24</y>
       <width>149</width>
       <height>149</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="circleBar_2">
      <property name="geometry">
       <rect>
        <x>-25</x>
        <y>-25</y>
        <width>199</width>
        <height>199</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QLabel" name="usedSize">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>58</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>X%</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>78</y>
        <width>100</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>X GB</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>219</y>
       <width>145</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>241</y>
       <width>145</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>263</y>
       <width>145</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="storage_11">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>195</y>
       <width>145</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Fast Access</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
    </widget>
    <widget class="QLabel" name="storage_12">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>285</y>
       <width>145</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>More Disk Information</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="specs">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>370</y>
      <width>515</width>
      <height>358</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>0</y>
       <width>446</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Machine's Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>22</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>45</y>
       <width>400</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TEMP</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>23</y>
       <width>400</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>CPU</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>44</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>67</y>
       <width>400</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TEMP</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>66</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="usageframe">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>105</y>
       <width>503</width>
       <height>247</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="usage">
      <property name="geometry">
       <rect>
        <x>-7</x>
        <y>-5</y>
        <width>516</width>
        <height>257</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>85</y>
       <width>446</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>CPU usage in the past minute</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="settings">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>370</y>
      <width>515</width>
      <height>358</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>43</y>
       <width>440</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Basic Disk Informations Section - Locations of Shortcuts to Check</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>76</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string>Pictures</string>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>76</y>
       <width>360</width>
       <height>20</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>60</y>
       <width>115</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>151</x>
       <y>60</y>
       <width>324</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Location</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>101</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string>Documents</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>101</y>
       <width>360</width>
       <height>20</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>126</y>
       <width>360</width>
       <height>20</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>126</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string>Downloads</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>0</y>
       <width>446</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Basic settings</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_4">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>19</y>
       <width>100</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Dark Mode</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_5">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>19</y>
       <width>325</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Pin Window</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clearTrashButton_2">
     <property name="geometry">
      <rect>
       <x>254</x>
       <y>330</y>
       <width>126</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset Settings</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clearTrashButton_3">
     <property name="geometry">
      <rect>
       <x>384</x>
       <y>330</y>
       <width>126</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply Changes</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="tasks">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>5</y>
      <width>515</width>
      <height>358</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="taskManager">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>520</width>
       <height>358</height>
      </rect>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>740</y>
      <width>84</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>X GB </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="storage_7">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>757</y>
      <width>84</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>X GB </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="storage_3">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>757</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Downloads</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="storage_4">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>740</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Documents</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="storage_6">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>750</y>
      <width>84</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>X GB </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="storage_2">
    <property name="geometry">
     <rect>
      <x>314</x>
      <y>750</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Pictures</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1045</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuStorage">
    <property name="title">
     <string>Disks</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuSpecs">
    <property name="title">
     <string>Usage</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTask_Manager">
    <property name="title">
     <string>Tasks</string>
    </property>
   </widget>
   <addaction name="menuStorage"/>
   <addaction name="menuSpecs"/>
   <addaction name="menuTask_Manager"/>
   <addaction name="menuSettings"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
